### ISSUE TITLE
Update module with approved verbs

### ISSUE THREAD

**Issue by DarkLite1** [nohwnd/Assert#30]
Fist of all, I would like to thank you for this amazing extension on `Pester`. This module made my life a lot easier, especially thanks to the `Assert-Equivalent` CmdLet. 

After updating the module from the gallery with `Update-Module Assert`, we receive the following warning messages:

```
PS C:\Users\me> Import-Module Assert -Verbose
VERBOSE: Loading module from path 'C:\Program Files\WindowsPowerShell\Modules\Assert\0.9.0\Assert.psd1'.
VERBOSE: Populating RepositorySourceLocation property for module Assert.
VERBOSE: Loading module from path 'C:\Program Files\WindowsPowerShell\Modules\Assert\0.9.0\Assert.psm1'.
WARNING: The names of some imported commands from the module 'TypeClass' include unapproved verbs that might make them
less discoverable. To find the commands with unapproved verbs, run the Import-Module command again with the Verbose
parameter. For a list of approved verbs, type Get-Verb.
WARNING: The names of some imported commands from the module 'TypeClass' include unapproved verbs that might make them
less discoverable. To find the commands with unapproved verbs, run the Import-Module command again with the Verbose
parameter. For a list of approved verbs, type Get-Verb.
```
These are probably just some `Should be` that should be `Should -Be` 😄 in version 0.9.0. 

PS: On a side note, when searching in Google for `Pester Assert module` your GitHub page doesn't show up. I think it deserves some better advertisement as it's a real life saver.

**Comment by nohwnd** [comment]
Hello, those warnings are coming from Is- in the TypeClass submodule I am using. I forgot to ignore the names in the latest release. Will patch when I get home.

**Comment by DarkLite1** [comment]
No big deal, just thought I'd give you a heads up

**Comment by nohwnd** [comment]
Fixed by [0.9.1](https://github.com/nohwnd/Assert/releases/tag/0.9.1)

**Comment by DarkLite1** [comment]
Thank you @nohwnd great work 👍
